Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM maven:3.5-jdk-8
+
+ARG MAVEN_OPTS="-Xmx1024M -Xss128M -XX:MetaspaceSize=512M -XX:MaxMetaspaceSize=1024M -XX:+CMSClassUnloadingEnabled"
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+
+RUN mvn clean package -DskipTests
+RUN chmod +x /usr/src/app/bin/pkg/*.sh
+
+ENTRYPOINT [ "./uReplicator-Distribution/target/uReplicator-Distribution-pkg/bin/start-controller.sh" ]
```

Comment:
Shall we have the same docker image for both controller and server?
Better to have a new start-ureplicator.sh and you can specify role[controller/server] there.